api/contracts/IERC721Metadata.json

```diff
@@ -330,6 +330,23 @@
       "outputs": [],
       "stateMutability": "nonpayable",
       "type": "function"
+    },
+    {
```

Comment:
it would be better to move this to another file if possible. you just need to copy the structure of this one and remove everything except this.